nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -113,7 +113,7 @@ ALLOWED_URL_SCHEMES = (
 CACHEOPS_DEFAULTS = {"timeout": int(os.getenv("NAUTOBOT_CACHEOPS_TIMEOUT", "900"))}
 
 # Set to False to disable caching with cacheops. (Default: True)
-CACHEOPS_ENABLED = is_truthy(os.getenv("NAUTOBOT_CACHEOPS_ENABLED", "True"))
+CACHEOPS_ENABLED = is_truthy(os.getenv("NAUTOBOT_CACHEOPS_ENABLED", "False"))
```

Comment:
So users with existing `nautobot_config.py` will still be defaulting to `True` unless they regenerate their configs. Can we call that out in the release overview that this is something they may need to manually change when upgrading?

Reply:
Yes.